### ISSUE TITLE
No module named 'liquid.utils' when creating new Environment

### ISSUE THREAD

**Issue by MarcelPa** [jg-rp/liquid#1]
Hey there,

I am trying to use your package to parse some liquid template strings without a "real" environment. When creating a new Environemnt like
```python
from liquid import Environment
env = Environment()
```
I get the error message from the title:
`ModuleNotFoundError: No module named 'liquid.utils'`

I installed the library using pipenv: `pipenv install python-liquid`. When checking the virtualenv created by pipenv, the `utils` module indeed is not installed:
```bash
site-pacakges $ ls liquid
ast.py  context.py  environment.py  exceptions.py  expression.py  filter.py  __init__.py  lex.py  loaders.py  mode.py  parse.py  __pycache__  stream.py  tag.py  template.py  token.py
```

I assume the `setup.py` does [not explicitly include](https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html?highlight=find_packages#using-find-or-find-packages) the submodules needed in its packages (i.e. it does not include `liquid.utils`).

Is my assumption correct or did I install the package wrong?

**Comment by jg-rp** [comment]
Hi,

Apologies. I misunderstood the behaviour of `setuptools.find_packages`. And I was only running the test suit with the package installed in dev mode.

Fixed in version 0.5.6, which has been uploaded to PyPi. So `pipenv update python-liquid` should do the trick.

Cheers,

**Comment by MarcelPa** [comment]
Works like a charm, thanks for the quick update :-)
